docs/open-source-project/release-notes/tizen-2-4.md

```diff
@@ -880,44 +883,44 @@ New Features
 - Open source upgrade
   - EFL version has been upgraded from 1.7 to 1.13.
 - API changes
-  - Eina_Bool edje_text_class_get (const char *text_class, char **font, Evas_Font_Size *size) has been changed to  Eina_Bool edje_text_class_get (const char *text_class, const char **font, Evas_Font_Size *size).
-  - void elm_win_wm_rotation_preferred_rotation_set(Evas_Object *obj, const int rotation) has been changed to void elm_win_wm_rotation_preferred_rotation_set(const Evas_Object *obj, int rotation).
+  - Eina_Bool edje_text_class_get (**const char text_class**, **char font**, **Evas_Font_Size** size) has been changed to  Eina_Bool edje_text_class_get (**const char text_class**, **const char font**, **Evas_Font_Size** size).
```

Comment:
Just copy below line and paste at line 886.
```
Eina_Bool edje_text_class_get (const char \*text_class, char \*\*font, Evas_Font_Size \*size) has been changed to Eina_Bool edje_text_class_get (const char \*text_class, const char \*\*font, Evas_Font_Size \*size).
```

Reply:
Done